design_handoff_mission_control/prototype/v1-dark.jsx

```
@@ -0,0 +1,825 @@
+            boxShadow:`0 0 18px ${D.uv}88`,
+          }}>
+            <OrbitStar size={16} color="#fff" />
+          </div>
+          <span style={{ fontWeight:600, fontSize:14, letterSpacing:'-0.01em', color:D.ink }}>Mission Control</span>
+        </div>
+        <span style={{ color:D.fgMuted, fontFamily:D.mono, fontSize:11, letterSpacing:'0.06em' }}>WED · 2026-05-27 · 09:12</span>
+
+        <nav style={{ marginLeft:18, display:'flex', gap:4, alignItems:'center' }}>
+          {['overview','insights','review'].map(t => (
+            <button key={t} onClick={() => setTab(t)} style={{
+              padding:'5px 12px', fontSize:12, fontWeight:500, textTransform:'capitalize',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)